trulens_eval/examples/snowflake_demo/custom_feedback/small_local_models.py

```diff
@@ -11,7 +11,7 @@
 
 CONTEXT_RELEVANCE_MODEL_PATH = os.getenv(
     "SMALL_LOCAL_MODELS_CONTEXT_RELEVANCE_MODEL_PATH",
-    "/trulens_demo/small_local_models/context_relevance",
```

Comment:
you should just set this env variable locally so you don't have to change this (this is just the default value) as this is the location of the file in the docker image.